Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -6497,6 +6497,13 @@ void WebPageProxy::didFailProvisionalLoadForFrameShared(Ref<WebProcessProxy>&& p
         } else
 #endif
             callClientFunctions();
+    } else {
+        if (auto* websitePolicies = navigation ? navigation->websitePolicies() : nullptr; websitePolicies
+            && (websitePolicies->advancedPrivacyProtections().contains(AdvancedPrivacyProtections::HTTPSFirst) || m_preferences->httpSByDefaultEnabled())
```

Comment:
Should the capitalization of httpSByDefaultEnabled be httpsByDefaultEnabled?

Reply:
Yes, but I'll need to adjust the generator script, and I'm going to remove this pref soon, so I decided against making that change.